setup.py

```diff
@@ -34,7 +34,8 @@ def _get_version():
     ["ml-pipelines-sdk>=0.26.3<2.0.0", "ml-metadata>=0.26<2.0.0"],
     "schema_curation": [
         "tfx>=0.26.3<2.0.0",
-    ]
+    ],
+    "xgboost_evaluator": ["xgboost==1.4.2"]
```

Comment:
nit: we should make this constraint wider to accept more versions of xgboost unless we use internal APIs.